### PR TITLE
[AR] Finished translation of week07-2

### PR DESCRIPTION
**Pull request by mraafat89**

### CODE REVIEW

**Comment by AliEl-01** [comment]
hi i saw your PR (07-2) it's perfect! Bravo!
but i have a point that i noticed and i see i need to get your attention!
anyway :
1- the title is not translated !
2- the term "Fig", it's better to translate it into "الشكل"
3- I noticed pargraphs even if they are translated! they stilll displayed in English (above the Arabic translation) on the page!!!

**Comment by AliEl-01** [comment]
1
title

**Comment by AliEl-01** [comment]
2
fig

**Comment by AliEl-01** [comment]
3
para
.
paragraph

**Comment by mraafat89** [comment]
Thanks for your comments. I fixed all of them

**Comment by AliEl-01** [comment]
Suuuuuuuper 👌🏻

**Review by AliEl-01 — APPROVED** [review]

File: docs/_config.yml [hunk 1]
```diff
@@ -455,6 +455,9 @@ ar:
         - path: ar/week06/06-1.md
         - path: ar/week06/06-2.md
         - path: ar/week06/06-3.md
+    - path: ar/week07/07.md
```
Comment (Atcold) [inline docs/_config.yml:458]: There is no such file.

**Review by Atcold — COMMENTED** [review]
`lang: ar` was missing, I've added it for you.
You need to submit `07.md` as well, if you're including it in `docs/_config.yml`.

**Review by Atcold — CHANGES_REQUESTED** [review]
You need to edit `docs/_config.yml` or provide `07.md`.

File: docs/_config.yml (hunk 1 above)
Comment (AliEl-01) [reply docs/_config.yml:458]: hi, i hope this PR #587 fix this !

Comment (Atcold) [reply docs/_config.yml]: Sure, but you cannot include stuff that does not exist.

**Review by Atcold — APPROVED** [review]
Removed unexisting reference from `docs/_config.yml` and added `lang: ar` myself.
Now it should be good to go.

**Pull request merged**